tools/recetox_aplcms/recetox_aplcms_recover_weaker_signals.xml

```diff
@@ -0,0 +1,128 @@
+<tool id="recetox_aplcms_recover_weaker_signals" name="RECETOX apLCMS - recover weaker signals" version="@TOOL_VERSION@+galaxy0">
+    <description>recover weaker signals from LC/MS spectra</description>
+    <macros>
+        <import>macros.xml</import>
+        <import>macros_split.xml</import>
+    </macros>
+    <expand macro="creator"/>
+
+    <expand macro="requirements"/>
+    <command detect_errors="aggressive"><![CDATA[
+        sh ${symlink_inputs} &&
+        Rscript -e 'source("${__tool_directory__}/utils.R")' -e 'source("${__tool_directory__}/main.R")' -e 'source("${run_script}")'
+    ]]></command>
+    <configfiles>
+        <configfile name="symlink_inputs">
+            #for $infile in $ms_files
+                ln -s '${infile}' '${infile.element_identifier}'
```

Comment:
I suspect here we are running in the same issue with having inputs that are not collections (since we use multiple="true"). Details on this arehere: https://planemo.readthedocs.io/en/latest/writing_advanced.html#processing-lists-reductions

Reply:
It seems to be working fine, probably multiple files are treated as a collection and `element_identifier` is available.

Reply:
I think this can then be resolved?